inbox/api/jsc_api.py

```diff
@@ -2,15 +2,17 @@
 import urllib
 import requests
 import simplejson
+import json
```

Comment:
I think we don't need this since we're importing `simplejson` in the line above.